modules/s2i/bash/artifacts/usr/local/s2i/assemble

```diff
@@ -8,9 +8,40 @@ source "${JBOSS_CONTAINER_MAVEN_S2I_MODULE}/maven-s2i"
 # include our s2i_core_*() overrides/extensions
 source "${JBOSS_CONTAINER_JAVA_S2I_MODULE}/s2i-core-hooks"
 
+# include our jlink scripts
+source "${JBOSS_CONTAINER_JAVA_JLINK_MODULE}/mkdeps.sh"
```

Comment:
This order seems odd. The order in `runall.sh` was

1. mkdeps.sh
2. mkstrippeddeps.sh
3. generatejdkdeps.sh
4. mkjreimage.sh